pkg/redis/PhpRedis.php

```diff
@@ -82,6 +82,14 @@ public function connect()
                 );
             }
 
+            if (array_key_exists('pass', $this->config)) {
+                $this->config['auth'] = $this->config['pass'];
+            }
+
+            if (array_key_exists('auth', $this->config)) {
+                $this->redis->auth($this->config['auth']);
+            }
+
```

Comment:
not sure we need the first if `if (array_key_exists('pass', $this->config)) {`.

```php
<?php
            if (array_key_exists('pass', $this->config)) {
                $this->redis->auth($this->config['pass']);
            }
```

Reply:
This is needed for services like Herold.
They provide a token for authentication in the password part.

Reply:
The code I posted does exactly the same but with one `if` statement.